scss/sections/trade/_trade.scss

```diff
@@ -109,18 +83,23 @@
                 font-size: 16px;
                 margin-left: 5%;
                 margin-bottom: 5%;
+                font-weight: 400;
             }
         }
         select {
             display: none;
         }
     }
-    .secondary_header {
-    justify-content: center;
-    p {
-        color: black;
+    .white_section {
```

Comment:
looks like more global selector. only for section so not dedicated just to trade.